app/models/organisation_provider.rb

```diff
@@ -0,0 +1,13 @@
+# == Schema Information
+#
+# Table name: organisation_provider
+#
+#  id              :integer          not null, primary key
+#  provider_id     :integer
+#  organisation_id :integer
+#
+
+class OrganisationProvider < ApplicationRecord
```

Comment:
If this is just a join table I don't think you need or want this model, it's just noise.